install/kubernetes/cilium/templates/clustermesh-apiserver/service.yaml

```
@@ -1,4 +1,4 @@
-{{- if and .Values.clustermesh.useAPIServer (eq .Values.clustermesh.apiserver.kvstoremesh.kvstoreMode "internal") }}
+{{- if and .Values.clustermesh.useAPIServer (eq .Values.clustermesh.apiserver.kvstoremesh.kvstoreMode "internal") (not .Values.clustermesh.apiserver.service.enabled) }}
```

This condition seems to be incorrectly negated, I would suggest doing the following:
```suggestion
{{- if and .Values.clustermesh.useAPIServer (eq .Values.clustermesh.apiserver.kvstoremesh.kvstoreMode "internal") .Values.clustermesh.apiserver.service.enabled }}
```